From 953d03acbbb7a8a5b8cb66acfb2ae3a49e6685a0 Mon Sep 17 00:00:00 2001
From: Robert Marko <robimarko@gmail.com>
Date: Sat, 10 May 2025 13:35:51 +0200
Subject: [PATCH] init: dts: include <linux/platform_device.h>

"struct platform_device" definition is currently missing, so include
the required kernel header.

Signed-off-by: Robert Marko <robimarko@gmail.com>
---
 src/init/ssdk_dts.c | 1 +
 1 file changed, 1 insertion(+)

--- a/src/init/ssdk_dts.c
+++ b/src/init/ssdk_dts.c
@@ -38,6 +38,7 @@
 #include <linux/of.h>
 #include <linux/of_mdio.h>
 #include <linux/of_platform.h>
+#include <linux/platform_device.h>
 
 static ssdk_dt_global_t ssdk_dt_global = {0};
 #ifdef HPPE
